py2many/cli.py

```diff
@@ -315,6 +315,42 @@ def _relative_to_cwd(absolute_path):
     return pathlib.Path(os.path.relpath(absolute_path, CWD))
 
 
+def _format_one(settings, output_path, env=None):
+    restore_cwd = False
```

Comment:
@jayvdb the test failures are due to "format.jl/ktlint" not available. But why only on windows?

I suspect it has something to do with the `os.chdir` logic in this function. Perhaps it's better to write:

```
try: 
  ...
finally:
    os.chdir(restore_cwd)
```

Reply:
I thought about it some more. Earlier we had a try-catch around `_process_one()`. After this change, we have a try-catch only around `transpile_one()`. So any exceptions from `format_one()` is getting propagated and logged as a test failure.

I added a try-catch around `format_one()` to maintain the same behavior as the old code.

However, we need to look into why windows has significantly fewer test passes vs ubuntu. I suspect it's setup problems with formatters.

Reply:
It is a problem with invoking the formatters on Windows.  Its got to do with .bat wrappers, `COMSPEC`, and oddities about win32 exec.  I've been down that rabbit hole once, and didnt get close enough to a neat solution to get a PR up.